### DISCUSSION TITLE
Can you add support for removing items and accessing parent / children items?

### DISCUSSION THREAD

**Discussion by DarVoid** [fyne-io/systray#60] [Ideas]
We were trying to create a systray app to poll if we had new PRs [this project](https://github.com/DarVoid/git_appletini)
but for that we needed to be able to remove items from the menu, so we gave it a go, and It was working for some time, but then I have no idea what changed (possibly some dependency) and we couldn't use it anymore.

Could you try to add the remove functionality?

Check[ this commit history](https://github.com/DarVoid/systray/commits/master) from a fork for further context

**Comment by andydotxyz** [comment]
In the meantime you can clear the menu and set a new one, it will look the same to your users I think.
